packages/crypto-wallet-core/src/validation/bch/index.ts

```diff
@@ -3,21 +3,25 @@ const BitcoreCash = require('bitcore-lib-cash');
 
 export class BchValidation implements IValidation {
   validateAddress(network: string, address: string): boolean {
+    const AddressCash = BitcoreCash.Address;
+    // Regular Address: try Bitcoin Cash
+    return AddressCash.isValid(address, network);
+  }
+
+  validateUri(network: string, address: string): boolean {
     // Check if the input is a valid uri or address
     const URICash = BitcoreCash.URI;
     const AddressCash = BitcoreCash.Address;
 
     // Bip21 uri
     let uri, uriAddress;
-    if (/^bitcoincash:|^bchtest:i/.test(address)) {
+    if (/^bitcoincash:|^bchtest:/i.test(address)) {
```

Comment:
Can we use bitcore-lib-cash for this?

Same thing as here:
https://github.com/bitpay/bitcore/pull/2421/files#diff-57413206f34f99cbc83e311f3b62cde2R11

but with bitcore-lib-cash.